README.Rmd

```diff
@@ -32,6 +32,8 @@ license](https://img.shields.io/badge/License-MIT-blue.svg)](https://github.com/
 ```{r, results = "asis", echo=FALSE}
 cat(gsub("\n[ ]+", " ", packageDescription("baselinenowcast")$Description))
 ```
+The below figure depicts the low-level function workflow, depicting options for estimating delays and uncertainty from other reporting triangles. ![](./man/figures/function_flow.png)
```

Comment:
I'm not really sure we want this here? Especially as this is really the low level interface. IMO I see this as part of the low level vignette (that doesn't exist yet).

Reply:
Yes saw your comment, can remove.